tests/routes/helpers.test.js

```diff
@@ -0,0 +1,31 @@
+const { getPsrRequestedConvictions } = require('../../server/routes/helpers')
+
+const testConvictions = require('../../mappings/community/convictions/D541487-convictions.json').response.jsonBody.convictions
```

Comment:
There's a risk that if this file changes then your unit tests will start failing, despite the fact the code is still working as intended. Would it be better to create the testConvictions locally to fully isolate the unit test?